modules/machine-adding-aws-compute-cloudformation.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * machine_management/user_infra/adding-aws-compute-user-infra.adoc
+
+[id="machine-adding-aws-compute-cloudformation_{context}"]
+= Adding more compute machines to your AWS cluster by using CloudFormation templates
+
+You can add more compute machines to your {product-title} cluster on Amazon Web Services (AWS) that you created by using the sample CloudFormation templates.
+
+[IMPORTANT]
+====
+The CloudFormation template creates a stack that represents one compute machine. You must create a stack for each compute machine.
+====
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your compute nodes, you must review the provided information and manually create the infrastructure. If your cluster does not initialize correctly, you might have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You installed an {product-title} cluster by using CloudFormation templates and have access to the JSON file and CloudFormation template that you used to create the compute machines during cluster installation.
+* You installed the AWS CLI.
+
+.Procedure
+
+. Create another compute stack.
+.. Launch the template:
++
+[IMPORTANT]
+====
+You must enter the command on a single line.
+====
++
+----
+$ aws cloudformation create-stack --stack-name <name> <1>
```

Comment:
missing `\` at the end of line:
`$ aws cloudformation create-stack --stack-name <name> \ <1>`